components/array-object-table/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "@govuk-frederic/array-object-table",
+  "version": "0.0.1",
+  "dependencies": {
+    "@govuk-frederic/table": "^0.0.3",
+    "@govuk-frederic/utils": "^0.0.1"
+  },
+  "devDependencies": {
+    "@govuk-react/storybook-components": "^0.2.4",
```

Comment:
is this really right?
i.e. `govuk-react`?

Reply:
I'm not sure if that is right, actually, but that's how it is for all the components in Frederic and that's where that package is. I'm not sure if there should be a duplicate of it in Frederic or if it should be elsewhere.